packages/key-did-resolver/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "key-did-resolver",
-  "version": "2.0.4",
+  "version": "2.0.6",
```

Comment:
That is not right. Please, rebase to `develop`.

Reply:
I changed the version to 2.1.4. I was not fully aware of the synatax of semantic versioning.
I used these links as a guide: https://docs.npmjs.com/cli/v6/using-npm/semver and  https://semver.org/.

If 2.0.4 is better, let me know.

Reply:
Oh, we do not change versions in PR at all. That can lead to weird behaviour of Lerna right after merge. Please, move it back to the original version.

Reply:
Okay, I reverted to the original version.

Reply:
Does this look reasonable @ukstv ?

Reply:
I'm not sure why it is still showing merge conflicts. I'll have to check.

Reply:
Okay, the merge conflicts are resolved and the tests pass. I'm getting a linter that is very fussy, but this could be that mine was out of date and I tried to update it.